app/component/map/map.scss

```diff
@@ -88,14 +88,32 @@ div.current-location-marker > span > svg.icon {
 }
 
 div.leaflet-marker-icon.from, div.leaflet-marker-icon.to {
-  > span > svg.icon {
-    width: 24px;
-    height: 24px;
-    stroke: white;
-    stroke-width: 60px;
+  > span {
+    &::before {
+      color: white;
+      position: absolute;
+      left: 6.4pt;
```

Comment:
For mac chrome and firefox:
left: 6.0pt;
top: 0pt;

works better. what do you think?
<img width="73" alt="screen shot 2015-12-03 at 14 01 50" src="https://cloud.githubusercontent.com/assets/1298522/11560098/a5eadb2e-99c6-11e5-9037-cda15d20a1dc.png">